### DISCUSSION TITLE
Landing pull requests - what's the process?

### DISCUSSION THREAD

**Discussion by KrayzeeKev** [axios/axios#5179] [General]
What is the process for PRs to be landed? There are approved PRs out there that fix real problems in 1.1.3 and it's unclear what has to happen and who has to do it. There appears to be a mechanism for a maintainer to approve but is this purely a "time" thing - I know people are volunteers? Or has somebody got to do something to highlight to somebody else that something needs doing?

As it stands, I can't use 1.x because of various header issues, one of which breaks axios-retry. And my security department is going to come after me with a vengeance for running 0.27 when 1.1 is out (and it's automated - I am out of date and vulnerable to hacking BY DEFINITION). So I'm keen to get somebody's PR landed so I can jump. 

The alternative of removing Axios is looming and I REALLY don't want that to happen.

**Comment by jasonsaayman** [comment]
sorry may have been a lul in doing reviews but i am back at it now so they will be merged on a more regular cadence
